apps/editing-toolkit/package.json

```diff
@@ -16,73 +16,15 @@
-		"build:newspack-blocks": "NODE_ENV=production yarn run newspack-blocks",
-		"starter-page-templates": "BROWSERSLIST_ENV=wpcom calypso-build --env source='starter-page-templates'",
-		"dev:starter-page-templates": "yarn run starter-page-templates",
-		"build:starter-page-templates": "NODE_ENV=production yarn run starter-page-templates",
-		"event-countdown-block": "BROWSERSLIST_ENV=wpcom calypso-build --env source='event-countdown-block'",
-		"dev:event-countdown-block": "yarn run event-countdown-block",
-		"build:event-countdown-block": "NODE_ENV=production yarn run event-countdown-block",
-		"global-styles": "BROWSERSLIST_ENV=wpcom calypso-build --env source='global-styles'",
-		"build:global-styles": "NODE_ENV=production yarn run global-styles",
-		"dev:global-styles": "yarn run global-styles",
-		"global-styles-customizer-fonts": "BROWSERSLIST_ENV=wpcom calypso-build --env source='global-styles/customizer-fonts'",
-		"build:global-styles-customizer-fonts": "NODE_ENV=production yarn run global-styles-customizer-fonts",
-		"dev:global-styles-customizer-fonts": "yarn run global-styles-customizer-fonts",
-		"jetpack-timeline": "BROWSERSLIST_ENV=wpcom calypso-build --env source='jetpack-timeline'",
-		"dev:jetpack-timeline": "yarn run jetpack-timeline",
-		"build:jetpack-timeline": "NODE_ENV=production yarn run jetpack-timeline",
-		"wpcom-block-editor-nux": "BROWSERSLIST_ENV=wpcom calypso-build --env source='wpcom-block-editor-nux'",
-		"dev:wpcom-block-editor-nux": "yarn run wpcom-block-editor-nux",
-		"build:wpcom-block-editor-nux": "NODE_ENV=production yarn run wpcom-block-editor-nux",
-		"wpcom-block-editor-welcome-tour": "BROWSERSLIST_ENV=wpcom calypso-build --env source='wpcom-block-editor-welcome-tour'",
-		"block-inserter-modifications": "npm-run-all --parallel block-inserter-modifications-*",
-		"block-inserter-modifications-contextual-tips": "BROWSERSLIST_ENV=wpcom calypso-build --env source='block-inserter-modifications/contextual-tips'",
-		"dev:block-inserter-modifications": "yarn run block-inserter-modifications",
-		"build:block-inserter-modifications": "NODE_ENV=production yarn run block-inserter-modifications",
-		"wpcom-block-editor-nav-sidebar": "BROWSERSLIST_ENV=wpcom calypso-build --env source='wpcom-block-editor-nav-sidebar'",
-		"dev:wpcom-block-editor-nav-sidebar": "yarn run wpcom-block-editor-nav-sidebar",
-		"build:wpcom-block-editor-nav-sidebar": "NODE_ENV=production yarn run wpcom-block-editor-nav-sidebar",
-		"whats-new": "BROWSERSLIST_ENV=wpcom calypso-build --env source='whats-new'",
-		"dev:whats-new": "yarn run whats-new",
-		"build:whats-new": "NODE_ENV=production yarn run whats-new",
-		"dev": "node bin/npm-run-build.js --dev",
-		"build": "node bin/npm-run-build.js --build",
-		"postbuild": "yarn validate-es5",
+		"build-cmd": "yarn run clean && yarn sync:newspack-blocks --nodemodules && node bin/npm-run-build.js",
+		"dev": "export NODE_ENV=development && yarn build-cmd",
```

Comment:
If I just do `NODE_ENV=production`, then it does not persist through the call to build-cmd. Export solves that, but I'm hoping there's a better way.